src/shrutiDJun20CodeExams/CodeExam10.java

```diff
@@ -0,0 +1,68 @@
+package shrutiDJun20CodeExams;
+
+/*
+ Print minimum rotation required to compare both strings equality.
+ Example 1  : 
+ String input1 = "abcdef";
+ String input2 = "defabc";
+ output : 3
+ */
+import java.util.ArrayList;
+import java.util.Arrays;
+//import java.util.HashMap;
+//import java.util.Iterator;
+//import java.util.LinkedHashSet;
+
+public class CodeExam10 {
+
+	void rotateElemets(ArrayList<String> list1, ArrayList<String> list2) {
+
+		int count = 0;
+		ArrayList<String> tempList1 = new ArrayList<String>(list1);
+		if (list1.size() == list2.size()) {
+			for (String temp : list1) {
+				if (tempList1.equals(list2)) {
+					break;
+				} else {
+					temp = tempList1.remove(list1.size() - 1);
+					tempList1.add(0, temp);
+					count++;
+				}
+			}
+			System.out.println("No. Of rotation needed for inputs to match--> "
+					+ count);
+		} else {
+			System.out.println("Inputs provided are of diff length");
+		}
+	}
+
+	/*void rotateElementWithoutCollection(String input1, String input2) {
+		int count = 0;
+		if (input1.length() == input2.length()) {
+			while (!input1.equals(input2)) {
```

Comment:
Just a suggestion. what if String can't be made equals even after all possible rotation ?? this will run infinitely ...

you can handled it by adding one more condition.
while(!input1.equals(input2) && count< input1.length)